Sage/carbon
# fix(link): ensure button has a type when used - FE-5706

Link component when used as a button currently does not supply the button with a type. This has caused a bug when `Pager` is used in a `Form`. The bug was causing the `onPrevious` callback function in `Pager` to be called in an onClick event when the enter key pressed.
    
fixes #5675

### Proposed behaviour

When `Pager` is used in a `Form`, the `onPrevious` callback function should not be called when the `Enter` key is pressed. 

### Current behaviour

When `Pager` is used in a `Form`, the `onPrevious` callback function is called when the `Enter` key is pressed. 

### Checklist




#### QA


### Additional context

N/A

### Testing instructions

New story has been added to the test folder of Form called `default with pager`. This story should not call `onPrevious` when the Textbox of Pager is highlighted and enter is pressed.

Codeesandbox built from issue - https://codesandbox.io/s/carbon-quickstart-forked-phv98c
